plugins/modules/notification/mattermost.py

```diff
@@ -127,7 +139,8 @@ def main():
             text=dict(type='str'),
             channel=dict(type='str', default=None),
             username=dict(type='str', default='Ansible'),
-            icon_url=dict(type='str', default='https://www.ansible.com/favicon.ico'),
+            icon_url=dict(type='str', default=None),
+            icon_emoji=dict(type='str', default=None),
```

Comment:
`default=None` is the default.
```suggestion
            icon_url=dict(type='str'),
            icon_emoji=dict(type='str'),
```